srcDoc/release_notes.rst

```diff
@@ -798,6 +807,33 @@ we could get into the following error:
 Known Issues
 ------------
 
+Running into SHMEM error on CPP Modern
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The default transport for *RTI Perftest* is `UDPv4` + `SHMEM`.
+In the case that the user is running a test on the same machine, thus using
+Shared Memory (`SHMEM`), and using CPP Modern. The user may get those errors
+message when running the application:
+
```

Comment:
*RTI Perftest* uses `UDPv4` + `SHMEM` by default. Certain Operative Systems don't support Shared Memory, or the default configuration is not enough for *RTI Connext DDS* to work properly. In those cases *RTI Perftest* will show some errors trying to create the Participant entity:

Reply:
Done